src/main/java/de/hysky/skyblocker/utils/Utils.java

```diff
@@ -441,11 +472,6 @@ private static void parseLocRaw(String message) {
         if (locRaw.has("map")) {
             map = locRaw.get("map").getAsString();
         }
-
-        if (locationChanged) {
-            SkyblockEvents.LOCATION_CHANGE.invoker().onSkyblockLocationChange(location);
-            locationChanged = false;
-        }
```

Comment:
Why did you only remove the event invocation?

Reply:
The event invocation was removed because the Mod API should (or will if its working fully) trigger it, other mods which still use locraw would result in this event triggering twice which would break the mod's functionality that it fires only once (when swapping worlds) such as with the egg finder.